26/generated/connect_config.html

```diff
@@ -375,7 +375,7 @@ <h4><a id="admin.listeners" href="#admin.listeners">admin.listeners</a></h4>
 <table><tbody>
 <tr><th>Type:</th><td>list</td></tr>
 <tr><th>Default:</th><td>null</td></tr>
-<tr><th>Valid Values:</th><td>org.apache.kafka.connect.runtime.WorkerConfig$AdminListenersValidator@7b3300e5</td></tr>
+<tr><th>Valid Values:</th><td>org.apache.kafka.connect.runtime.WorkerConfig$AdminListenersValidator@d2cc05a</td></tr>
```

Comment:
This is just some bug in the connect configs/docs, filed https://issues.apache.org/jira/browse/KAFKA-12700

Reply:
Thanks for dealing with this. Looks like a community member already submitted a PR, so we'll fix that shortly.